package.json

```diff
@@ -9,20 +9,20 @@
   "exports": {
     ".": {
       "types": "./dist/index.d.ts",
-      "require": "./dist/index.js",
-      "import": "./dist/index.mjs"
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js",
+      "default": "./dist/index.js"
     }
   },
-  "main": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.mts",
   "files": [
     "dist",
     "src",
     "README.md"
   ],
   "scripts": {
-    "build": "nanobundle build",
-    "clean": "nanobundle clean",
+    "build": "tsdown",
```

Comment:
For what I understood, clean script is not used in repository. I think it's an equivalence of the add clean option.